DrinkManagerWeb/Views/Shared/_Layout.cshtml

```diff
@@ -52,22 +57,18 @@
                         <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Reviewed drinks</a>
                     </li>
                 </ul>
-            </div>
-
-            <main class="col-10 p-0">
+            </nav>
 
+            <main class="col-10 pt-5">
                 <!-- page body -->
-                
-            @RenderBody()
-               
-
-                <!-- footer -->
-                <footer class="border-top footer text-muted">
-                    <div class="p-1 pl-4">
-                            &copy; 2020 - DrinkManagerWeb
-                        </div>
-                    </footer>
+                @RenderBody()
             </main>
+            <!-- footer -->
+            <footer class="footer text-muted">
+                <div class="p-1 pl-4">
+                    &copy; 2020 - DrinkManagerWeb
```

Comment:
When you moved this text on the left side of website, this whole footer is unnecessary in my opinion. There is nothing on it, on main website and on the others sites.